docs/sources/operations/multi-tenancy.md

```diff
@@ -17,3 +17,19 @@ To run in multi-tenant mode, Loki should be started with `auth_enabled: true`.
 Loki can be run in "single-tenant" mode where the `X-Scope-OrgID` header is not
 required. In single-tenant mode, the tenant ID defaults to `fake`.
 
+## Multi-tenant Queries
+
+If run in multi-tenant mode, queries across different tenants can be enabled via
+`multi_tenant_queries_enabled: true` option in the querier. Once enabled multiple
+tenant IDs can be defined in the HTTP header `X-Scope-OrgID` by concatenating them
+with `|`. For instance a query for tenant A and B can set `X-Scope-OrgID: A|B`.
```

Comment:
That didnt work.
When use `X-Scope-OrgID=onetenant|twotenant` loki response 200 but empty data.

You must use `X-Scope-OrgID` x2 like that:
![image](https://user-images.githubusercontent.com/29627495/165939512-7d88e984-c37f-4413-951e-b7b8956c9778.png)

Reply:
Inserting. The piping worked for me. Let me investigate. Thanks for trying it out.

Reply:
Ok, I've tried it with the Docker compose setup in `tools/dev/loki-boltdb-storage-s3`. How did you try it out?

Reply:
```yaml
analytics:
  reporting_enabled: false
auth_enabled: true
common:
  path_prefix: /var/loki
  replication_factor: 1
  storage:
    s3:
      access_key_id: access
      bucketnames: bucketname
      endpoint: endpoint
      http_config:
        idle_conn_timeout: 90s
        insecure_skip_verify: false
        response_header_timeout: 0s
      region: region
      secret_access_key: secret
compactor:
  shared_store: aws
  working_directory: /var/loki/compactor
frontend:
  log_queries_longer_than: 20s
memberlist:
  join_members:
  - loki-memberlist
querier:
  multi_tenant_queries_enabled: true
schema_config:
  configs:
  - from: "2021-09-07"
    index:
      period: 24h
      prefix: loki_index_
    object_store: aws
    schema: v11
    store: boltdb-shipper
storage_config:
  aws:
    access_key_id: access
    bucketnames: bucketname
    endpoint: endpoint
    http_config:
      idle_conn_timeout: 90s
      insecure_skip_verify: false
      response_header_timeout: 0s
    region: region
    secret_access_key: secret
  boltdb_shipper:
    active_index_directory: /var/loki/index
    cache_location: /var/loki/index_cache
    shared_store: s3
tracing:
  enabled: true
```

So i found that when use simple-scale helm chart.


![image](https://user-images.githubusercontent.com/29627495/166635964-61c7c472-0e11-4246-b226-9c88835972ea.png)

![image](https://user-images.githubusercontent.com/29627495/166636308-521e10d5-40d8-4f20-8a1f-8b98ca05a560.png)

If you want i cant provide logs from grafana for answers loki.

Reply:
Seems like on my instance multi-tenant queries didnt work.
I need send __tenant_id__ from log collector? Or loki create that index selfs?

Reply:
What are the screenshots showing? What are tags here? `X-Scope-OrgID='default|system'
image` should be correct on the datasource. However, that also means Promtail is sending logs either as `default` or `system` tenant.

Reply:
Let me explain.


When i use `Loki` datasource and set `X-Scope-OrgID=default` or `X-Scope-OrgID=system` - we get that graph:
![image](https://user-images.githubusercontent.com/29627495/166739685-18f44f56-c361-48c5-a5b2-c22dadd61f4f.png)


But when i set `X-Scope-OrgID=default|system` i get "no data".

So my first screenshots just trace from loki where read service get 0 Series when `tenants_ids=[default|system]` and 3k when `tenants_ids=[default]` 



About promtail - already have tenants in loki and `X-Scope-OrgID` without `multitenant_query` work fine - just in code and your example i see `__tenant_id` label and didnt see that in my loki installation

Reply:
Thanks for your patience. I'm not really sure where you see `tenants_ids=[default|system]`.

Anyhow, which version of Loki are you using? It must be a recent main build after at least commit `[c7cbeab](https://github.com/grafana/loki/commit/c7cbeab8e2338b1dc39a3ff3bcde29af58317823)`.

Reply:
Yes - i found problem.

We didnt update master branch after release.
So right now everything fine.
Thanks for help!